arches/app/models/resource.py

```diff
@@ -1047,9 +1047,13 @@ def get_geom(self,buffer_distance=''):
         bool.must(Match(field='_all', query=self.entityid))
         query.add_query(bool)
         results = query.search(index='resource')
-        try:
-            geomjson = results['hits']['hits'][0]['_source']['geometry']
-        except:
+        hits = results['hits']['hits']
```

Comment:
@mradamcox any reason why you prefer this syntax to try/except?

Reply:
Just for clarity. Reading through the code now it's more clear why a None is returned. Try/except could trip up for other reasons, a KeyError on `hits`, `_source`, or `geometry` for example. Another way to have handled it would be `try:/except IndexError:` which would, in this case only trip if there were zero results in the return. This could work, but I felt like it would still be easier to debug if the circumstances were very clearly laid out.